integration-test/992-boundaries-min_zoom-and-name.py

```diff
@@ -127,13 +127,15 @@ def test_10m_regions(self):
 class BoundariesMinZoomAndNameOsm(FixtureTest):
     def test_region_boundary_zug_luzern(self):
         # Switzerland region HAS name, OpenStreetMap
+        # do this at z12, as the boundary between Zug and Luzern is quite
+        # short, and we want enough space to label.
         self.load_fixtures([
             'http://www.openstreetmap.org/relation/1686447',
             'http://www.openstreetmap.org/relation/1685677',
-        ], clip=self.tile_bbox(8, 133, 89))
+        ], clip=self.tile_bbox(12, 2144, 1438))
 
         self.assert_has_feature(
-            8, 133, 89, 'boundaries',
```

Comment:
I think we should still keep the zoom 8 test but for `kind: region` only as that proves we are taking the OSM data correctly at zoom 8.

Reply:
Good idea. Fixed in fb5565c.